### PR TITLE
ability to pass configuration to /buildtarget

### PR DESCRIPTION
**Pull request by jchannon**

### CODE REVIEW

File: OmniSharp/Build/BuildTargetRequest.cs [hunk 1]
```diff
@@ -6,12 +6,20 @@ public class BuildTargetRequest : Request
     {
         public BuildType Type { get; set; }
 
+        public BuildConfiguration Configuration { get; set; }
```
Comment (nosami) [inline OmniSharp/Build/BuildTargetRequest.cs]: This needs to default to Debug

Comment (jchannon) [inline OmniSharp/Build/BuildTargetRequest.cs]: It will, just like Type does to Build because they're enums

**Comment by nosami** [comment]
Yep, of course. Cheers!

**Comment by jchannon** [comment]
Got this working but I think I am going to have to do another PR so you can do `/p:Platform="Any CPU"`

**Comment by nosami** [comment]
Think you're probably going to need to query for build targets thinking
about it.

**Pull request merged**